test/coverage-test/test/allow-external.test.ts

```diff
@@ -4,28 +4,41 @@ import { readCoverageMap, runVitest, test } from '../utils'
 test('{ allowExternal: true } includes files outside project root', async () => {
   await runVitest({
     include: ['fixtures/test/allow-external-fixture.test.ts'],
-    coverage: { allowExternal: true, reporter: 'json', include: ['**/fixtures/**'], all: false },
+    coverage: {
+      allowExternal: true,
+      reporter: 'json',
+      include: ['**/fixtures/src/math.ts', '**/test/test-utils/fixtures/**'],
+      all: true,
+    },
   })
   const coverageMap = await readCoverageMap()
   const files = coverageMap.files()
 
   // File outside project root
   expect(files).toContain('<project-root>/test/test-utils/fixtures/external-math.ts')
 
+  // Uncovered files outside project root should also be included
+  expect(files).toContain('<project-root>/test/test-utils/fixtures/uncovered.ts')
```

Comment:
This is actually working correctly already. Uncovered files are resolved from the project root:

https://github.com/vitest-dev/vitest/blob/8ac70111706a4444a779a75bed34c18b800ea0b3/packages/coverage-v8/src/provider.ts#L373-L386

https://github.com/vitest-dev/vitest/blob/8ac70111706a4444a779a75bed34c18b800ea0b3/packages/coverage-istanbul/src/provider.ts#L399-L412